### ISSUE TITLE
Enhancement: Sort or List Specific Genre

### ISSUE THREAD

**Issue by opensourcefan** [zigurana/EmulationStation#16]
It would be very helpful if it was possible to sort by Genre or better yet just pick one specific one to show. For example going into the options and selecting puzzle and having all the puzzle games show.

**Comment by zigurana** [comment]
Its a good idea, and it is something that has been on the radar for quite a while now. The main reason for having a proper database rather than a XML list for storing the game-lists in. As you can see, there is a specific branch called 'gamelistdb' even in the original Aloshi repo. I think @Herdinger or @t3hi3x might take it up at some point.
Specifically for use on the RPi, it is not a done deal if this is feasible, performance-wise. Does open up a whole range of new ways to cut through these otherwise unwieldy large collections of ROMs though, so its certainly nice to have.
Long story short, not something I will pick up anytime soon, but keep an eye on the other forks!
